content/en/docs/concepts/certificaterequest.md

```diff
@@ -72,170 +97,498 @@ Each ready condition consists of the pair `Ready` - a boolean value, and
+the CertificateRequests that are referencing either built-in types or external
+issuer types (such as the
+[`GoogleCASIssuer`](https://github.com/jetstack/google-cas-issuer) or the
+[`AWSPCAIssuer`](https://github.com/jniebuhr/aws-pca-issuer) type).
+
+Let us imagine that our approver needs to approve CertificateRequests that are
+targeting the built-in ACME issuer as well as the `AWSPCAIssuer`:
 
 ```yaml
-apiGroups: ["cert-manager.io"]
-resources: ["signers"]
-verbs: ["approve"]
-resourceNames:
- # namesapced signers
- - "<signer-resource-name>.<signer-group>/<signer-namespace>.<signer-name>"
- # cluster scoped signers
- - "<signer-resource-name>.<signer-group>/<signer-name>"
- # all signers of this resource name
- - "<signer-resource-name>.<signer-group>/*"
-```
-
-An example ClusterRole that would grant the permissions to set the Approve and
-Denied conditions of CertificateRequests that reference the cluster scoped
-`myissuers` external issuer, in the group `my-example.io`, with the name `myapp`:
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-issuer
+  namespace: some-namespace
+spec:
+  acme:
+    email: jane.doe@gmail.com
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: letsencrypt-issuer-account
+    solvers:
+      - http01:
+          ingress:
+            class: traefik
+---
+apiVersion: awspca.cert-manager.io
+kind: AWSPCAClusterIssuer
+metadata:
+  name: awspca-issuer
+  namespace: some-namespace
+spec:
+  arn: "pca-807911e9"
+```
+
+To make our approver work for all CertificateRequests across all namespaces, we
+apply the following role to our cluster:
 
 ```yaml
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: my-example-io-my-issuer-myapp-approver
+  name: opa-approver-for-acme-and-aws-pca
+rules:
+  - apiGroups: ["cert-manager.io"]
+    resources: ["signers"]
+    verbs: ["approve"]
+    resourceNames:
+      - "clusterissuers.acme.cert-manager.io/letsencrypt-issuer"
+      #  <-------------------------------->  <---------------->
+      #    resource type + resource group        issuer name
+      - "awspcaclusterissuers.awspca.cert-manager.io/awspca-issuer"
+```
+
+Note: the "resource type"
+[is](https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology)
+the lower-case and plural version of the "kind" field.
+
+
+### Syntax of the `resourceNames` field used in RBAC roles {#approval-api-rbac-syntax}
+
+When a user or controller attempts to approve or deny a CertificateRequest, the
+cert-manager webhook evaluates whether it has sufficient permissions to do so.
+These permissions are based upon the request itself, and more specifically, the
+CertificateRequest's `issuerRef` field.
+
+{{% pageinfo color="warning" %}}
+
+The verb `approve` on the resource `signer` only works with `ClusterRole`, and
+the `apiGroup` must be set to `cert-manager.io`.
```

Comment:
This seems quite random and these two things don't feel like they should be in the same sentence. Let's keep the first part and change it to: The approver needs the approve permissions at the cluster level.

The group is obvious since we are interacting with cert-manager.